string destination
string location
string start_time
string boarding_time
string arrival_time
